carla_ego_vehicle/config/sensors.json

```diff
@@ -7,7 +7,7 @@
             "width": 800,
             "height": 600,
             "fov": 100,
-            "sensor_tick": 0.05,
+            "^": 0.05,
```

Comment:
why did we change `sensor_tick` to `^` ?

Reply:
@arkadiy-telegin

Reply:
Seems like a typo. Fixed.